docs/linux/coverage.md

```diff
@@ -15,18 +15,124 @@ ln -s `which mips64el-linux-gnuabi64-readelf` ~/bin/mips64le/readelf
 export PATH=~/bin/mips64le:$PATH
 ```
 
-### objdump
+The target-triple prefix is determined based on the _target_ config option.
+
+### readelf
+
+`readelf` is used to detect virtual memory offset. Executor truncates PC values into `uint32` before sending them to `syz-manager` and `syz-manager` has to detect the offset.
```

Comment:
This part needs some clarifications. What are virtual memory offsets? Addresses of basic blocks? Functions? Something else?

Reply:
This information was in original document, expanded now with the newest commit